docs/real-device-config.md

```diff
@@ -189,6 +188,28 @@ If this was successful, the output should end with something like:
     }
 ```
 
+#### Build for Generic iOS/tvOS devices, run manually
+
+As another way, you can build `WebDriverAgentRunner` for a generic iOS/tvOS device, and install the generated `.app` package to a real device.
+The generic one means for any iOS/tvOS devices similar to creating a release iOS/tvOS package.
```

Comment:
Building for a generic device means the package could be installed on any iOS or tvOS device depending on the chosen platform.